apps/src/util/dismissVersionRedirect.js

```diff
@@ -0,0 +1,38 @@
+const dismissVersionRedirect = module.exports;
+
+// A session variable storing a comma-delimited list of course/script names for which:
+// ...the user has already dismissed the version redirect warning.
+const DISMISSED_REDIRECT_WARNINGS_SESSION_KEY = 'dismissedRedirectWarnings';
+// ...the user has already dismissed the version redirect dialog (<RedirectDialog/> component).
+const DISMISSED_REDIRECT_DIALOGS_SESSION_KEY = 'dismissedRedirectDialogs';
+
+dismissVersionRedirect.dismissedRedirect = function (sessionKey, name) {
+  const dismissedRedirects = sessionStorage.getItem(sessionKey);
+  return (dismissedRedirects || '').includes(name);
```

Comment:
it looks like this could accidentally match substrings. it would be better to do `return (dismissedRedirects || '').split(',').includes(name);`